### PR TITLE
[Vision] add test cases for flip, normalize, to_tensor

### PR DESCRIPTION
**Pull request by javelinjs**
https://github.com/apache/incubator-mxnet/issues/8556

### CODE REVIEW

File: python/mxnet/gluon/data/vision/transforms.py [hunk 1]
```diff
@@ -184,26 +184,26 @@ def forward(self, x):
         return image.imresize(x, *self._args)
 
 
-class RandomHorizontalFlip(HybridBlock):
-    """Randomly flip the input image horizontally with a probability
+class RandomLeftRightFlip(HybridBlock):
```
Comment (piiswrong) [inline python/mxnet/gluon/data/vision/transforms.py]: FlipLeftRight?

File: tests/python/unittest/test_gluon_data_vision.py [hunk 2]
```diff
@@ -18,22 +18,54 @@
...
+
+def test_random_left_right_flip():
+    def f_expect(img):
+        pil_img = Image.fromarray(img).transpose(Image.FLIP_LEFT_RIGHT)
+        return np.array(pil_img)
+    run_random(transforms.RandomLeftRightFlip(),
+        f_expect,
+        np.random.uniform(0, 255, (300, 300, 3)).astype(dtype=np.uint8))
+
+def test_random_top_bottom_flip():
```
Comment (piiswrong) [inline tests/python/unittest/test_gluon_data_vision.py]: How about run tests on non random versions (flip_left_right, adjust_xxx)

**Comment by javelinjs** [comment]
I'll add tests for brightness/contrast... once this pr is approved

File: python/mxnet/gluon/data/vision/transforms.py [hunk 3]
```diff
@@ -184,26 +184,44 @@ def forward(self, x):
         return image.imresize(x, *self._args)
 
 
-class RandomHorizontalFlip(HybridBlock):
-    """Randomly flip the input image horizontally with a probability
+class FlipLeftRight(HybridBlock):
```
Comment (piiswrong) [inline python/mxnet/gluon/data/vision/transforms.py]: randomflipxxx

Comment (piiswrong) [inline python/mxnet/gluon/data/vision/transforms.py]: we don't need this for now. It's unclear when you would want to always flip

Comment (javelinjs) [reply python/mxnet/gluon/data/vision/transforms.py]: fixed

**Comment by marcoabreu** [comment]
Hello @javelinjs, please rebase your PR.

**Comment by javelinjs** [comment]
@marcoabreu You mean rebase from master? Is there anything related to these operators?

**Pull request merged**